watch-site.js

```diff
@@ -0,0 +1,54 @@
+const chokidar = require('chokidar')
+const bs = require("browser-sync").create()
+const generator = require('mini-book-site-generator')
+const Lock = require('./lock.js')
+const processorLock = new Lock()
+
+const antoraArgs = ['--playbook', 'site.yml']
+
+const watcher = chokidar.watch([
+  `src/docs/**`,
+], {
+  ignored: [
+    /(^|[\/\\])\../, // ignore dotfiles
+  ],
+  persistent: true,
+  // avoid issues with Intellij IDEA temporary files (used to automatically save changes)
+  // [Error: ENOENT: no such file or directory, lstat '/path/to/.../file.adoc~' in /path/to/... (ref: antora-local-dev <worktree> | path: modules/...)] {
+  //   errno: -2,
+  //   code: 'ENOENT',
+  //   syscall: 'lstat',
+  //   path: '/path/to/.../file.adoc~'
+  // }
```

Comment:
Can this section be deleted?

Reply:
It's a note for our future selves to explain why we need to add/configure `awaitWriteFinish`. In my opinion, we should keep it but if you insist, I can remove it.